README.md

```diff
@@ -1,4 +1,5 @@
 # Datomic Pro Starter
+[![Codefresh build status]( https://g.codefresh.io/api/badges/pipeline/parkside/docker-datomic%2Fbuild?type=cf-1&key=eyJhbGciOiJIUzI1NiJ9.NWQyMjViNjU3NDY0Mzk5M2U3N2JiNTEy.-giiapIOw102lhi-hws0d7U6jlpwDCUp2xObdY08hzk)]( https://g.codefresh.io/pipelines/edit/new/builds?id=60de11e46483e097c22ff814&pipeline=build&projects=docker-datomic&projectId=60de11e26483e078e72ff811)
```

Comment:
I think it is okay to put the badge in the README of the public repo as the Codefresh project itself is protected.